### ISSUE TITLE
New metadata editor UI

### ISSUE THREAD

**Issue by WrinklyNinja** [loot/loot#547]
Due to performance issues encountered as part of #434, it has become necessary to move the plugin metadata editor UI out of the card list, so that it is not continually recycled while scrolling cards. This necessarily restricts the editor to being a single element, so that only one editor can be open at any one time - opening an editor would simply grey out the option to do so in plugin card menus.

One possibility would be to have the editor display as a sheet that pops up underneath the card list and sticks to the bottom of the screen, eg:

![screenshot 12](https://cloud.githubusercontent.com/assets/2430738/13536756/7c046fbc-e239-11e5-8a84-a03b607ad05f.png)

Alternatively, the editor could be displayed over the whole card list, hiding it completely, eg.

![screenshot 12 - copy](https://cloud.githubusercontent.com/assets/2430738/13537175/7fc32286-e23b-11e5-82f3-0e5be0f142cd.png)

In this case the "Jump to General Information" and Search buttons would be greyed out as their effects would be invisible.

The choice between the two really depends on how useful it is to see other plugins' card content when editing metadata. The sidebar is retained in both cases because the priority info visibility is useful, as is drag 'n' drop of sidebar items into the editor tables.

Of course, more drastic changes to the UI for editing metadata are possible, but I can't think of what they might be.

@Freso: Do you have any ideas/opinions?

**Comment by WrinklyNinja** [comment]
I've started work on implementing the top screenshot, since it's the simplest solution and is at least better than nothing. It currently looks like this:

![screenshot 13](https://cloud.githubusercontent.com/assets/2430738/13548741/3338bcb4-e2ef-11e5-9924-a8a28569b12d.png)

I changed the background to light blue because the relatively heavy purplish background is no longer necessary to help differentiate editor cards from normal-state plugin cards, and I think the lighter colour looks 'fresher'.

I've also removed the CRC, version info and plugin state icons from the editor toolbar as that info is all visible in the plugin's card and it's not worth duplicating.

Still to do:
- [ ] Limit the maximum height of the editor to 50% of the height available to it and the plugin list, and have the content below its toolbar scroll if necessary.
- [ ] Improve the styling of the tables
- [ ] Disabling the Edit Metadata menu option if the editor is already open - at the moment any existing editor content is overwritten when the menu option is selected.
